### ISSUE TITLE
Proper default value support

### ISSUE THREAD

**Issue by brendt** [spatie/data-transfer-object#41]
If you've set a default value, you'll still get an error when not passing the value to the construct.

```php
class InvoiceCollection extends DataTransferObject
{
    /** @var array */
    public $invoices = [];
}
```

```php
// This throws an uninitialised error

$invoiceCollection = new InvoiceCollection();
```

**Comment by sasa-b** [comment]
I'd be willing to help, I'm going to fork and make a PR if that's OK with you :)

**Comment by brendt** [comment]
Sure! Looking forward to it

**Comment by brendt** [comment]
Fixed with 1.6.5: https://github.com/spatie/data-transfer-object/releases/tag/1.6.5
